docs/tutorials/longer-metrics-storage.md

```diff
@@ -56,9 +53,10 @@ size` and `dbengine disk space`.
 metrics.
 
 Based on our testing, these default settings will retain about two day's worth of metrics when Netdata collects 2,000
```

Comment:
With the new slab collectors that have been added to the agent I believe it's closer to 4,000 metrics now.

Reply:
Will the default settings will still store two days worth of metrics @ 4,000 metrics, or does this need to change to something like:

> Based on our testing, these default settings will retain about one day's worth of metrics when Netdata collects 4,000 metrics every second.

Reply:
It will be one day now.

Reply:
Changed. Thank you!